datadog-service-catalog.yml

```diff
@@ -0,0 +1,491 @@
+  - name: va-mobile-app
+    type: slack
+    contact: https://dsva.slack.com/archives/C018V2JCWRJ
+links:
+  - name: Mobile API Dashboard 2.0
+    type: dashboard
+    provider: Datadog
+    url: https://vagov.ddog-gov.com/dashboard/j6m-3ws-x4c/mobile-api-dashboard-20?refresh_mode=sliding&from_ts=1697647938386&to_ts=1698252738386&live=true
+  - name: Mobile API Dashboard
+    type: dashboard
+    provider: Datadog
+    url: https://vagov.ddog-gov.com/dashboard/pzz-7x2-4cs/mobile-api-dashboard
+integrations: {}
+---
+schema-version: v2.1
+dd-service: claim-status
+team: benefits
+application: claim status tool application
+tier: '1'
+description: Claim Status Tool application
+lifecycle: production
+contacts:
+  - name: benefits-management-tools
+    type: slack
+    contact: https://dsva.slack.com/archives/C04KHCT3ZMY
+links:
+  - name: Benefits - Claim Status Tool
+    type: dashboard
+    provider: Datadog
+    url: https://vagov.ddog-gov.com/dashboard/uc8-jnr-zhm/benefits---claim-status-tool
+integrations: {}
+---
+schema-version: v2.1
+dd-service: gibill-application
+team: vfs-education
+tier: '1'
+lifecycle: production
+contacts:
+  - name: benefits-education
+    type: slack
+    contact: https://dsva.slack.com/archives/CG6N59X40
+integrations: {}
+---
+schema-version: v2.1
+dd-service: healthcare-application
+team: 1010-health-apps
+application: health
+tier: '1'
+lifecycle: production
+contacts:
+  - name: 1010-health-apps
+    type: slack
+    contact: https://dsva.slack.com/archives/CMJ2V70UV
+links:
+  - name: 1010 Health Apps
+    type: dashboard
+    provider: Datadog
+    url: https://vagov.ddog-gov.com/dashboard/p5g-fys-epz/1010-health-apps
+integrations: {}
+---
+schema-version: v2.1
+dd-service: profile
+team: auth-experience-profile
+tier: '1'
+lifecycle: production
+contacts:
+  - name: accountexp-authexp
+    type: slack
+    contact: https://dsva.slack.com/archives/C909ZG2BB
+integrations: {}
+---
+schema-version: v2.1
+dd-service: education-forms
+team: vfs-education
+tier: '1'
+lifecycle: production
+contacts:
+  - name: benefits-education
+    type: slack
+    contact: https://dsva.slack.com/archives/CG6N59X40
+integrations: {}
+---
+schema-version: v2.1
+dd-service: ask-va
+team: octo-engineers
+tier: '2'
+lifecycle: production
+contacts:
+  - name: ask-va-public
+    type: slack
+    contact: https://dsva.slack.com/archives/C05A2F6DEAE
+integrations: {}
+---
+schema-version: v2.1
+dd-service: evidence-upload
+team: benefits
+application: Notice of Disagreement, Supplemental Claims
+tier: '1'
+description: Evidence upload services for Notice of Disagreement and Supplemental
+  Claims
+lifecycle: Production
+contacts:
+  - name: benefits-management-tools
+    type: slack
+    contact: https://dsva.slack.com/archives/C04KHCT3ZMY
+links:
+  - name: Benefits - Evidence Uploads
+    type: dashboard
+    provider: Datadog
+    url: https://vagov.ddog-gov.com/dashboard/u78-itq-2yt
+integrations: {}
+---
+schema-version: v2.1
```

Comment:
let's wipe this test-10-10-cg service - it was something Aparna was trying, as a mapping from what's in the VFS application directory. But I don't like having individuals emails in there and it conflicts with the `caregiver-application` service.